.builders/upload.py

```diff
@@ -103,6 +116,8 @@ def main():
             print(f'{prefix} Name: {project_metadata["Name"]}')
             print(f'{padding}Version: {project_metadata["Version"]}')
 
+            sha256_digest = hash_file(wheel)
```

Comment:
Can we move this back down closer to where we use it like before?

Reply:
I had to move it because we use it in two places now, namely this new reference here: https://github.com/DataDog/integrations-core/blob/0584b450f8ab8a9d706f6903bc5bc6362cbe771b/.builders/upload.py#L136